src/dcd_mapping/align.py

```diff
@@ -13,7 +13,11 @@
 from Bio.SearchIO._model import Hit, QueryResult
 from cool_seq_tool.schemas import Strand
 
-from dcd_mapping.lookup import get_chromosome_identifier, get_gene_location
+from dcd_mapping.lookup import (
+    _get_normalized_gene_response,
```

Comment:
if we want to use this function here, we should rename it to remove the leading underscore, so that it's treated as "public" (see https://peps.python.org/pep-0008/#method-names-and-instance-variables, https://stackoverflow.com/a/55525523)